app/views/layouts/external_landing.html.erb

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
This is for registration links, eventually it may be used for renewals when we get around to harmonizing look and feel between them.